qualtran/bloqs/factoring/_factoring_shims.py

```diff
@@ -30,8 +46,21 @@ class MeasureQFT(Bloq):
     def signature(self) -> 'Signature':
         return Signature([Register('x', QBit(), shape=(self.n,), side=Side.LEFT)])
 
-    def decompose_bloq(self) -> 'CompositeBloq':
-        raise DecomposeTypeError('MeasureQFT is a placeholder, atomic bloq.')
+    def build_composite_bloq(self, bb: 'BloqBuilder', x: Soquet) -> Dict[str, 'SoquetT']:
```

Comment:
since this shim has a decomposition now should it become a bloq ?

Reply:
I'll leave that up to @mpharrigan. We added this decomposition a few PRs back (or meant to), but it got clobbered when merging two separate PRs touching this file (my mistake). I think the goal eventually is to use the phase_estimation bloqs, but for now I just added this so it could decompose entirely into leaf bloqs.